repos/pc/src/lib/wifi/firmware.cc

```diff
@@ -45,6 +45,8 @@ Firmware_list fw_list[] = {
 	{ "iwlwifi-9000-pu-b0-jf-b0-34.ucode", 2678284, nullptr },
 	{ "iwlwifi-9000-pu-b0-jf-b0-36.ucode", 2678284, "iwlwifi-9000-pu-b0-jf-b0-34.ucode" },
 	{ "iwlwifi-9000-pu-b0-jf-b0-46.ucode", 1514876, nullptr },
+	{ "iwlwifi-9260-th-b0-jf-b0-34.ucode", 2678092, nullptr },
+	{ "iwlwifi-9260-th-b0-jf-b0-46.ucode", 2678092, "iwlwifi-9260-th-b0-jf-b0-34.ucode" },
```

Comment:
This contradicts the change above. You define *iwlwifi-9260-th-b0-jf-b0-46.ucode* is equivalent to *iwlwifi-9260-th-b0-jf-b0-34.ucode*, which will then be requested as ROM. Also, [cnuke/](https://github.com/cnuke/dde_linux_firmware.git) does not contain *iwlwifi-9260-th-b0-jf-b0-46.ucode* on the topic branch.